src/bvar/detail/sampler.cpp

```diff
@@ -108,6 +109,7 @@ class SamplerCollector : public bvar::Reducer<Sampler*, CombineSampler> {
     void run();
 
     static void* sampling_thread(void* arg) {
+        butil::PlatformThread::SetName("brpc_sampling");
```

Comment:
In addition to the sampling thread, bvar also has a grab thread and two dump threads. (see variable.cpp and collector.cpp) Can you also add names to them?

Reply:
done